### PR TITLE
[WIP] Augur ui v2

### PR DESCRIPTION
**Pull request by petong**

### CODE REVIEW

File: packages/augur-ui/Dockerfile [hunk 1]
```diff
@@ -1,79 +1,22 @@
...
-COPY . /augur
-COPY support/local-run.sh /augur/local-run.sh
-
-# workaround a bug when running inside an alpine docker image
-RUN rm -f /augur/yarn.lock
 
 RUN set -ex; \
     if [ "$BUILD_ENVIRONMENT" = "dev" ]; then \
-        ETHEREUM_NETWORK=$ethereum_network yarn build --dev --augur-hosted --disableMainnet; \
+        ETHEREUM_NETWORK=$ethereum_network yarn workspace @augurproject/ui build --dev --augur-hosted --disableMainnet; \
```
Comment (justinbarry) [inline packages/augur-ui/Dockerfile]: Is `ETHEREUM_NETWORK` already defined? If so, we will need to install git above.

File: packages/augur-ui/Dockerfile [hunk 2]
```diff
@@ -1,79 +1,22 @@
...
-        ETHEREUM_NETWORK=$ethereum_network yarn build --production --augur-hosted --disableMainnet; \
+        ETHEREUM_NETWORK=$ethereum_network yarn workspace @augurproject/ui build --production --augur-hosted --disableMainnet; \
     elif [ "$BUILD_ENVIRONMENT" = "release" ]; then \
-        ETHEREUM_NETWORK=$ethereum_network yarn build --production --augur-hosted; \
+        ETHEREUM_NETWORK=$ethereum_network yarn workspace @augurproject/ui build --production --augur-hosted; \
     fi;
 
-# need arg to pass in for augur-ui (production) and augur-dev (dev)
-RUN git rev-parse HEAD > /augur/build/git-hash.txt \
-  && git log -1 > /augur/build/git-commit.txt \
```
Comment (justinbarry) [inline packages/augur-ui/Dockerfile:50]: Do we still want git-commit?

**Review by justinbarry — CHANGES_REQUESTED** [review]
A few comments.

File: packages/augur-ui/Dockerfile (hunk 2 above)
Comment (petong) [reply packages/augur-ui/Dockerfile:50]: I believe the addition of git-revision-webpack-plugin supersedes that command.

**Review by justinbarry — APPROVED** [review]

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**